### PR TITLE
feat: issue #95 added hero section

### PR DESCRIPTION
**Pull request by os-hi**
feat: added responsive homepage

### CODE REVIEW

File: src/routes/(default)/+page.svelte
Comment (directormac) [inline src/routes/(default)/+page.svelte:1]: Please export the component to the designated barrel file.

**Review by directormac — CHANGES_REQUESTED** [review]
No screenshot included for the output how can i know that the task is done or no indication of what  task is done on the issue.

File: src/routes/(default)/+page.svelte [hunk 1]
```diff
@@ -1,20 +1,8 @@
...
-		>
-	</h1>
-	<h1 class="h1">
-		<span
-			class="bg-gradient-to-br from-pink-500 to-violet-500 bg-clip-text text-transparent box-decoration-clone"
-			>Deploy.</span
-		>
-	</h1>
+<script>
+	import Hero from "@components/sections/hero.svelte";
```
[the remark this answers is not in the document]
Comment (os-hi) [reply src/routes/(default)/+page.svelte:2]: image
like this?

[the remark this answers is not in the document]
Comment (directormac) [reply src/routes/(default)/+page.svelte:2]: @os-hi

**Comment by os-hi** [comment]
desktop view
image
mobile view
image

**Pull request merged**